robottelo/ui/computeresource.py

```diff
@@ -137,10 +135,107 @@ def update(self, name, newname=None, parameter_list=None,
         self.click(common_locators['submit'])
 
     def delete(self, name, really=True):
-        """Removes the compute resource entity"""
+        """Removes the compute resource entity."""
         self.delete_entity(
             name,
             really,
             locators['resource.delete'],
             drop_locator=locators['resource.dropdown'],
         )
+
+    def access_compute_resource(self, res_name):
+        """Access a particular compute resource."""
+        strategy, value = locators['resource.get_by_name']
+        resource = self.search(res_name)
+        if resource is None:
+            raise UINoSuchElementError(
+                'Could not find the resource {0}'.format(res_name))
+        self.click((strategy, value % res_name))
+
+    def list_vms(self, res_name):
+        """Lists vms of a particular compute resource.
+        Note: Currently lists only vms that show up on the first page.
+        """
+        self.access_compute_resource(res_name)
+        self.click(locators['resource.virtual_machines_tab'])
+        vm_elements = self.find_element(locators['resource.vm_list'])
```

Comment:
Doesn't `self.find_element` always return only one/first element? You can look on the find_elements from Selenium library if you like to have list of WebElements